### PR TITLE
removed unused variables

### PR DESCRIPTION
**Pull request by Daniyal98**
## Description

closes: #2486 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/10-structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [x] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/core/04-channel/keeper/handshake_test.go [hunk 1]
```diff
@@ -638,7 +638,6 @@ func (suite *KeeperTestSuite) TestChanCloseInit() {
 
 			// create channel in init
 			path.SetChannelOrdered()
-			err = path.EndpointA.ChanOpenInit()
```
Comment (colin-axner) [inline modules/core/04-channel/keeper/handshake_test.go:641]: Why was this removed?

Comment (crodriguezvega) [reply modules/core/04-channel/keeper/handshake_test.go:641]: I added it back and added as well a`suite.Require().NoError(err)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -109,6 +109,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [\#2434](https://github.com/cosmos/ibc-go/pull/2478) Removed all `TypeMsg` constants
 * (modules/core/exported) [#1689] (https://github.com/cosmos/ibc-go/pull/2539) Removing `GetVersions` from `ConnectionI` interface.
 * (testing) [\#2657](https://github.com/cosmos/ibc-go/pull/2657) Carry `ProposerAddress` through commited blocks. Allow `DefaultGenTxGas` to be modified.
+* [\#2486] (https://github.com/cosmos/ibc-go/pull/pull/2692) Removed unused variables from `core/04-channel/keeper/packet_test.go` and `core/04-channel/keeper/handshake_test.go`.
```
Comment (crodriguezvega) [inline CHANGELOG.md]: I think we don't need a changelog entry for this change. Since I can edit the file, I will remove it myself.

**Review by crodriguezvega — APPROVED** [review]
Thanks for this clean up, @Daniyal98. Since I was able to edit the files in your branch I took out the changelog entry since I think we don't need it in this case.

**Review by colin-axner — APPROVED** [review]
